ee/tabby-db/migrations/0022_github-provider.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE github_provider (
+    id INTEGER NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL,
+    github_url TEXT NOT NULL,
```

Comment:
```suggestion
```

This field is mostly used for private github deployment (github ee). we can skip it for now.